src/main/webapp/resources/core/css/dssp.css

```diff
@@ -0,0 +1,117 @@
+* {
+    padding: 0;
+    margin: 0;
+    outline: none;
+    box-sizing: border-box;
+}
+.tabs-wrapper {
+    margin-left: 50%;
+    padding: 2%;
+    transform: translate(-50%);
+    background-color: #DDD;
+}
+.form-buttons {
+    width: 40px;
+    height: 40px;
+    background-color: white;
+    display: inline-block;
+    border: #999 solid medium;
+    border-radius: 50%;
+    transition: 0.3s;
+}
+.form-buttons:hover {
+    transition: 0.3s;
+    transform: scale(1.05);
+}
+#b1:checked ~ #btn-1 {
+    background-color: #AAA;
+}
+#b1:checked ~ .tabs .tab-2{
+    display: none;
+}
+
+#b2:checked ~ #btn-2 {
+    background-color: #AAA;
+}
+#b2:checked ~ .tabs .tab-1{
+    display: none;
+}
+.break {
+    flex-basis: 100%;
+    height: 0;
+}
+.tab {
+    width: 100%;
+    height: auto;
+    font-family: "Century Gothic";
+    font-weight: bold;
+    font-size: 20px;
+    color: #AAA;
```

Comment:
Please leave the text color unchanged. Black, in my opinion, looks more harmonious.

Reply:
done